javascript/javascript2/week3/fastest_presser/main.js

```diff
@@ -0,0 +1,62 @@
+//let timeInSecond=0;
```

Comment:
great! played the game and it works pretty well, 

There were some things i noticed that were a bit funny, i can start multiple games at the same time.
this can be fixed by disabling the button like this, when the game starts
```
document.getElementById("buttonStart").enabled = false
```